addons/web/static/src/views/fields/text/text_field.js

```diff
@@ -95,6 +97,7 @@ export class TextField extends Component {
         textarea.style.height = "auto";
         const height = Math.max(this.minimumHeight, textarea.scrollHeight + heightOffset);
         Object.assign(textarea.style, previousStyle, { height: `${height}px` });
+        this.state.height = height;
```

Comment:
what about bypassing owl for this? It might be overkill to trigger a re-rendering just to set the height attribute